admin/class-boldgrid-backup-admin-db-import.php

```diff
@@ -150,8 +150,48 @@ public function import_lines( $lines ) {
 			return false;
 		}
 
-		/* phpcs:disable WordPress.DB.RestrictedClasses */
-		$db = new PDO( sprintf( 'mysql:host=%1$s;dbname=%2$s;', DB_HOST, DB_NAME ), DB_USER, DB_PASSWORD );
+        // Parameters to check the DB_HOST setup
```

Comment:
It looks like we may be duplicating code?

Is it possible simply call `get_connection_string()`?